src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -486,35 +485,86 @@ Transactor::apply()
 NotTEC
 Transactor::checkSign(PreclaimContext const& ctx)
 {
+    // do not check signature of inner batch txn
+    if (ctx.tx.isFlag(tfInnerBatchTxn))
+        return tesSUCCESS;
```

Comment:
This should only return `tesSUCCESS` if `SigningPubKey` and `TxnSignature` are empty and `Signers` doesn't exist

Reply:
partial fix: https://github.com/XRPLF/rippled/pull/5060/commits/97cce91b1220f20114dee8ce22cdc686e8c1b4ec

Should the `TxnSignature` be empty or not exist?

Reply:
Discussed in a meeting - we agreed it should not exist since that's what multisign does.